src/passes/PostEmscripten.cpp

```diff
@@ -80,6 +81,23 @@ struct PostEmscripten : public Pass {
         func->body = builder.makeConst(Literal(int32_t(sbrkPtr)));
         func->module = func->base = Name();
       }
+      // Apply the sbrk ptr value, if it was provided.
```

Comment:
It would be good if this comment explained what it means to apply the sbrk ptr value. I'm still not sure why the value needs to be written into memory.

Reply:
Sure, added more text there now.